config/__tests__/config.ts

```diff
@@ -32,6 +34,8 @@ import {
   PersonalAccessKeyAccount,
   PersonalAccessKeyAccount_DEPRECATED,
 } from '../../types/Accounts';
+import * as configFile from '../configFile';
+//import * as config_DEPRECATED from '../config_DEPRECATED';
```

Comment:
@joe-yeager, see my note. I've added the `config_DEPRECATED` tests for your review. Any help is appreciated!

Reply:
Resolving for now--Joe reviewed and left me a comment on Slack.